app/controllers/sign_up/passwords_controller.rb

```diff
@@ -28,17 +30,10 @@ def create
 
     private
 
-    def process_successful_confirmation
-      process_valid_confirmation_token
-      render_page
-    end
-
-    def render_page
-      render(
-        :new,
-        locals: { confirmation_token: @confirmation_token },
```

Comment:
AFAICT we weren't ever using this `locals` assignment, since the view references `@confirmation_token` directly. Locals are intended to be referenced without the `@`.